### ISSUE TITLE
Move Trekarta's data to less restrictive folder

### ISSUE THREAD

**Issue by ptzlion** [andreynovikov/trekarta#231]
It's convenient to sync all Trekarta's data files (additional maps, tracks and waypoints) with third-party sync app like Resilio Sync. On earlier Android the data was in ./Trekarta - a folder that is simply accessible by other apps. On Android 9 the data resides in folder ./Android/data/mobi.maptrek but is still accessible by other apps. However on Android 13 (and some prev versions) the folder ./Android/data is  protected folder and a sync app can not get access to Trekarta's data. Is there any way to move the data ack to ./Trekarta? Thank you.

**Comment by andreynovikov** [comment]
No, it's the restriction added in recent versions of Android. There is a slightly difficult and not user friendly option to implement this with interactive access requests but I'm not sure that Resilio Sync will be able to read such a folder either.

**Comment by ptzlion** [comment]
Total Commander is able to gain access to such folders. Resilio Sync is not. :-( That's why i'm looking for some way to move necessary folders somewhere closer to root of storage. How do other apps store their data not in ./Android/data...? Take a look at AlReader - this app create folder ./AlReader and store it's data there. And there is a folder ./AlReader/sync in which AlReader stores a bookmarks. And i do sync this folder with Resilio Sync.

**Comment by andreynovikov** [comment]
![Screenshot_20240126_185012_Google Play Store.jpg](https://github.com/andreynovikov/trekarta/assets/2943625/a83c54ab-aef0-4162-aa9b-2da4f7273395)

As I explained, this is prohibited in modern versions.
